Userland/Libraries/LibWeb/WebIDL/OverloadResolution.cpp

```diff
@@ -271,12 +271,12 @@ JS::ThrowCompletionOr<ResolvedOverload> resolve_overload(JS::VM& vm, IDL::Effect
         //     then remove from S all other entries.
         else if (value.is_object()
             && has_overload_with_argument_type_or_subtype_matching(overloads, i, [](IDL::Type const& type) {
-                   // FIXME: - a callback interface type
-                   // FIXME: - a dictionary type
-                   // FIXME: - a record type
                    if (type.is_object())
                        return true;
                    return false;
+                   dbgln("FIXME: a callback interface type");
+                   dbgln("FIXME: a dictionary type");
+                   dbgln("FIXME: a record type");
```

Comment:
These would preferably be above the return statements even though you consolidate and move the return statement in the next commit

Reply:
Thanks — fixed now. (FWIW, I think that was an artifact of me having used `git add -p` and manually edited the diff, and git confused — or I did…)